docs/components/movement-sensor/wheeledodometry.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Configure a wheeledodometry movement sensor"
+linkTitle: "wheeled-odometry"
+type: "docs"
+description: "Configure a wheeled-odometry movement sensor."
+images: ["/icons/components/imu.svg"]
+tags: ["movement sensor", "components", "movement sensor"]
+# SMEs: Rand, Martha
+---
+
+Configure a `wheeled-odometry` movement sensor to implement _wheeled odometry_ on your robot.
+
+_Wheeled odometry_ is the estimation of position, orientation, linear velocity, and angular velocity using the dimensions of a base.
+This model uses [encoders](/components/encoder/) from [position reporting motors](/components/motor/gpio/) to get an odometry estimate from a wheeled base.
+
+## Set-up requirements
+
+To prepare your robot, attach [encoders](/components/encoder/) to each of the position-reporting motors on your base to measure their rotation.
+
+- Configure your rover as a [wheeled base component](/components/base/wheeled/).
+- Configure each of these position-reporting motors [as `gpio` motor components](/components/motor/gpio/).
```

Comment:
`Position Reporting` refers to a motor property, any motor model can have this property. The gpio motor with an encoder is one such example of a position-reporting motor. The roboclaw is a position reporting motor with encoders, the drive is always position-reporting, but a gpio motor without an encoder is not. The wheeled odometry model depends on a motor that can report its own position, usually a motor with an encoder of some sort.

Does that make a bit more sense?

Reply:
That makes sense! I will try and update to reflect, will comment here when done / when it's ready for re-review

Reply:
@randhid Ok tried to update with that advice!